provisioning_client/tests/hsm_client_riot_ut/hsm_client_riot_ut.c

```diff
@@ -107,15 +112,18 @@ static char* umocktypes_stringify_RIOT_ECC_PRIVATE(const RIOT_ECC_PRIVATE* value
     }
     else
     {
-        int length = snprintf(NULL, 0, "{ %p }", value->data);
+        int length = snprintf(NULL, 0, "{ %d, %zd, %p }",
+            value->s, value->n, value->p);
         if (length < 0)
         {
             result = NULL;
         }
         else
         {
-            result = (char*)my_gballoc_malloc(length + 1);
-            (void)snprintf(result, length + 1, "{ %p }", value->data);
+            length++;
+            result = (char*)my_gballoc_malloc(length);
+            (void)snprintf(NULL, 0, "{ %d, %zd, %p }",
```

Comment:
```suggestion
            (void)snprintf(result, 0, "{ %d, %zd, %p }",
```

Reply:
Ctrl C, Ctrl V error, very nice catch! That sprintf wouldn't do anything.

Reply:
should also be (result,length,"format string")

Reply:
Fixed